tools/RAiDER/models/hrrrak.py

```diff
@@ -0,0 +1,15 @@
+from RAiDER.models.hrrr import HRRR
+
+class HRRRAK(HRRR):
+    def __init__(self):
```

Comment:
The validity period needs to be updated for HRRR Alaska. Confirm in the documentation what the first file is in the S3 bucket for alaska.

Reply:
Done.